TomTomSDKExamples/Examples/UseCase/Offline/OfflineNavigationContent.swift

```diff
@@ -0,0 +1,689 @@
+//  © 2023 TomTom NV. All rights reserved.
+//
+//  This software is the proprietary copyright of TomTom NV and its subsidiaries and may be
+//  used for internal evaluation purposes or commercial use strictly subject to separate
+//  license agreement between you and TomTom NV. If you are the licensee, you are only permitted
+//  to use this software in accordance with the terms of your license agreement. If you are
+//  not the licensee, you are not authorized to use this software in any manner and should
+//  immediately return or destroy it.
+
+// System modules
+import Combine
+import CoreLocation
+import Foundation
+import SwiftUI
+
+// GO SDK modules
```

Comment:
we don't use GoSDK in public communication:

```suggestion
// TomTom SDK modules
```

Reply:
I'd suggest double-checking with @ioanakanda-tomtom the best way to name.

Reply:
Updated!